File: Python/D20-W/5/5.py
# Write a program that accepts the lengths of three sides of a triangle as inputs. The 
# program output should indicate whether or not the triangle is a right triangle.
#  x2 + y2 = z2



def right_Triangle():
    x = int(input("Enter first side value: "))
    y = int(input("Enter second side value: "))

    sides_square = (x**2) + (y**2)
    third_side_square = (x+y)**2

    if(sides_square == third_side_square):
        print("This triangle is a right triangle")
    else:
        print("This triangle is not a right triangle")

right_Triangle()

File: C/D4/5/5.cpp
// Write a C program to for nested for loops.

#include <stdio.h>

int main(){
    int i,j;

    for(i = 1; i <= 3; i++){
        for(int j = 1; j <= 2; j++){
            printf("%d", i);
        }
        printf("\n");      
    }
}